cdk/drupal/drupal_stack.py

```diff
@@ -1284,6 +1285,61 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
         )
         sg_https_ingress.cfn_options.condition = certificate_arn_exists_condition
 
+        # codebuild
+        codebuild_transform_service_role = aws_iam.Role(
+            self,
+            "CodeBuildTransformServiceRole",
+            assumed_by=aws_iam.ServicePrincipal('codebuild.amazonaws.com'),
+            inline_policies={
+                "TransformRolePermssions": aws_iam.PolicyDocument(
+                    statements=[
+                        aws_iam.PolicyStatement(
+                            effect=aws_iam.Effect.ALLOW,
+                            actions=[
+                                'logs:CreateLogGroup',
+                                'logs:CreateLogStream',
+                                'logs:PutLogEvents'
+                            ],
+                            resources=['*']
+                        ),
+                        aws_iam.PolicyStatement(
+                            effect=aws_iam.Effect.ALLOW,
+                            actions=[
+                                's3:GetObject',
+                                's3:PutObject'
+                            ],
+                            resources=[
+                                "arn:aws:s3:::github-user-and-bucket-taskcatbucket-2zppaw3wi3sx/*"
+                            ]
+                        )
+                    ]
+                )
+            }
+        )
+        with open('drupal/scripts/codebuild_transform_project_buildspec.yml') as f:
+            codebuild_transform_project_buildspec = f.read()
+        codebuild_transform_project = aws_codebuild.CfnProject(
+            self,
+            "CodeBuildTransformProject",
+            artifacts=aws_codebuild.CfnProject.ArtifactsProperty(
+                type="CODEPIPELINE",
+            ),
+            # TODO: add kms encryption key
+            # encryption_key="",
```

Comment:
can we use the default AWS KMS key here?

Reply:
@dylanvaughn I'm not quite sure what you want here. From the console there are no customer-created KMS keys (our stack doesn't include one) and there a number of "default" keys specific to various aws services -- see screenshot. Let me know if we want to re-use a key for another service, or if we want to create one master key as part of the pending encryption ticket and reference it everywhere in the stack.

![aws-kms-screenshot](https://user-images.githubusercontent.com/17485/82887886-eb28c200-9f48-11ea-9939-b277a0303efe.png)

Reply:
Yeah let's address in a separate PR